_search-plugins/ubi/schemas.md

```diff
@@ -0,0 +1,223 @@
+---
+layout: default
+title: UBI index schemas
+parent: User Behavior Insights
+has_children: false
+nav_order: 5
+---
+
+# UBI index schemas
+
+The User Behavior Insights (UBI) data collection process involves tracking and recording the queries submitted by users as well as monitoring and logging their subsequent actions or events after receiving the search results. There are two UBI index schemas involved in the data collection process:
+* The [query index](#ubi-queries-index), which stores the searches and results.
+* The [event index](#ubi-events-index), which stores all subsequent user actions after the user's query. 
+
+## Key identifiers
+
+For UBI to function properly, the connections between the following fields must be consistently maintained within an application that has UBI enabled:
+
+- [`object_id`](#object_id) represents an ID for whatever object the user receives in response to a query. For example, if you search for books, it might be an ISBN code of a book, such as `978-3-16-148410-0`.
+- [`query_id`](#query_id) is a unique ID for the raw query language executed and the `object_id` values of the _hits_ returned by the user's query.  
+- [`client_id`](#client_id) represents a unique query source. This is typically a web browser used by a unique user.
+- [`object_id_field`](#object_id_field) specifies the name of the field in your index that provides the `object_id`. For example, if you search for books, the value might be `isbn_code`.
+- [`action_name`](#action_name), though not technically an ID, specifies the exact user action (such as `click`, `add_to_cart`, `watch`, `view`, or `purchase`) that was taken (or not taken) for an object with a given `object_id`.
+
+To summarize, the `query_id` signals the beginning of a unique search for a client tracked through a `client_id`. The search returns various objects, each with a unique `object_id`. The `action_name` specifies what action the user is performing and is connected to the objects, each with a specific `object_id`. You can differentiate between types of objects by inspecting the `object_id_field`.  
+
+Typically, you can infer the user's overall search history by retrieving all the data for the user's `client_id` and inspecting the individual `query_id` data. Each application decides what makes a search session by examining the data in the backend.
```

Comment:
The meaning of the second sentence is unclear. Do we mean "Each application determines what constitutes a search session by examining the backend data"?